_maps/yogstation/RandomRuins/LavaRuins/lavaland_surface_abductorterrorship.dmm

```diff
@@ -0,0 +1,3645 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"aa" = (
+/turf/template_noop,
+/area/lavaland/surface/outdoors)
+"ab" = (
+/turf/open/floor/plating/asteroid/basalt/lava_land_surface,
+/area/lavaland/surface/outdoors)
+"ac" = (
+/turf/open/lava/smooth/lava_land_surface,
+/area/lavaland/surface/outdoors)
+"ad" = (
+/turf/closed/indestructible/abductor{
```

Comment:
indestructible walls 👎 
and if you end up making them real alien walls, you'd have to remove 99% of the ruin's content, since there's a lot of things in there that players shouldn't have, like the alien's batons. I would honestly just axe this ruin, and maybe replace it with something else that's better for the ashwalker whatever